data-model/src/main/java/io/micronaut/data/model/query/builder/sql/Dialect.java

```diff
@@ -82,6 +96,15 @@ public final boolean supportsArrays() {
         return supportsArrays;
     }
 
+    /**
+     * Some databases support JSON view.
+     * @return true if database supports JSON view
+     * @since 4.0.0
+     */
+    public final boolean supportsJsonView() {
```

Comment:
What exactly is the Json view?

Reply:
This is going away from here. 
But for what I would use JsonView is annotation telling that table/view returned result in JSON column.
I wanted to use it, probably not the best way, for users to mark results from Json View (Oracle Duality View) meaning it can return json object.

Reply:
We can mark the repository methods if the entity is marked as a JsonView etc

Reply:
I removed `JsonView` now and stick to QueryResult annotation, reading json columns will use custom json column readers if available.